notes-app/app.js

```diff
@@ -0,0 +1,71 @@
+// npm packages:
+const yargs = require('yargs');
+
+// custom packages:
+const noteHelper = require('./notes.js')
```

Comment:
make a decision to either use singular or plural and continue to do so throughout the project.
using singular notation in certain places and plural notation in others would creep in discrepancies.

Reply:
done